monkey/infection_monkey/exploit/tools/utils.py

```diff
@@ -0,0 +1,8 @@
+from typing import Sequence
+
+from infection_monkey.i_puppet import TargetHost
+
+
+def should_attempt_exploit(host: TargetHost, exploitation_ports: Sequence[int]) -> bool:
```

Comment:
This poorly named. Whether or not all ports are closed may be only one criteria an exploiter uses to determine whether or not it should attempt exploitation.

Reply:
Also, we should probably unit test this.